src/Cli/dotnet/commands/dotnet-workload/install/LocalizableStrings.resx

```diff
@@ -356,6 +356,9 @@
   <data name="CannotSpecifyVersionOnCommandLineAndInGlobalJson" xml:space="preserve">
     <value>Cannot specify a particular workload version on the command line via --version or --from-history when there is already a version specified in global.json file {0}. To update the globally installed workload version, run the command outside of the path containing that global.json file or update the version specified in the global.json file and run "dotnet workload update."</value>
   </data>
+  <data name="CannotSpecifyVersionOnCommandLineAndFromHistory" xml:space="preserve">
+    <value>Cannot specify a particular workload version on the command line via both --version and --from-history.</value>
+  </data>
```

Comment:
It looks like this string isn't used at all.

Reply:
You're right—I ultimately merged it with CannotCombineOptions. Removed.